tests/component/task/test_timing.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+
+from nidaqmx.constants import AcquisitionType, Edge, Level, LineGrouping, Polarity
+from nidaqmx.task import Task
+
+
+@pytest.fixture()
+def sim_6535_di_single_line_task(task, sim_6535_device) -> Task:
+    """Gets DI task."""
+    task.di_channels.add_di_chan(
+        sim_6535_device.di_lines[0].name, line_grouping=LineGrouping.CHAN_FOR_ALL_LINES
+    )
+    return task
+
+
+def test___timing___cfg_handshaking___no_errors(
+    sim_6535_di_single_line_task: Task,
+):
+    sim_6535_di_single_line_task.timing.cfg_handshaking_timing(
+        AcquisitionType.FINITE, samps_per_chan=2000
+    )
```

Comment:
This should validate that samp_timing_type == SampleTimingType.HANDSHAKE, samp_quant_samp_mode == AcquisitionType.FINITE, and samp_quant_samp_per_chan == 2000.

Likewise, the other tests should validate these things.